services/otel-collector/deploy/app.yaml

```
@@ -0,0 +1,66 @@
+apiVersion: opentelemetry.io/v1beta1
+kind: OpenTelemetryCollector
+metadata:
+  name: otel
+  namespace: monitoring
+spec:
+  env:
+    - name: APPLICATIONINSIGHTS_CONNECTION_STRING
+      valueFrom:
+        secretKeyRef:
+          name: app-insights-connstring
+          key: connectionString
+  image: otel/opentelemetry-collector-contrib:0.122.0
```

Hard pull rate limits on dockerhub so we should use our cache instead
```suggestion
  image: altinncr.azurecr.io/docker.io/otel/opentelemetry-collector-contrib:0.122.0
```